### PR TITLE
test(client): namedConstraints @@unique @@id with and without name

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: packages/client/src/__tests__/types/namedConstraints/test.ts [hunk 1]
```diff
@@ -23,6 +25,25 @@ async function main() {
         },
       },
     })
+
+  // @@unique
+  const result3 = await prisma.atAtUnique.findUnique({
```
Comment (janpio) [inline packages/client/src/__tests__/types/namedConstraints/test.ts:30]: Should this be `AtAtUniqueNamed` maybe? THe `name` is on that model.

Comment (Jolg42) [reply packages/client/src/__tests__/types/namedConstraints/test.ts:30]: Yes indeed

Comment (do4gr) [reply packages/client/src/__tests__/types/namedConstraints/test.ts]: I committed the changes to have the tests rerun

**Review by do4gr — APPROVED** [review]
Apart from the issue @janpio  detected this looks good to me.

File: packages/client/src/__tests__/types/namedConstraints/test.ts [hunk 2]
```diff
@@ -23,6 +25,25 @@ async function main() {
         },
       },
     })
+
+  // @@unique
+  const result3 = await prisma.AtAtUniqueNamed.findUnique({
```
Comment (janpio) [inline packages/client/src/__tests__/types/namedConstraints/test.ts:30]:
```suggestion
  const result3 = await prisma.atAtUniqueNamed.findUnique({
```

**Pull request merged**